Kickstarter-iOS/Features/ConfirmDetails/Views/ConfirmDetailsContinueCTAView.swift

```diff
@@ -56,37 +56,45 @@ final class ConfirmDetailsContinueCTAView: UIView {
   override func bindStyles() {
     super.bindStyles()
 
-    _ = self
-      |> \.layoutMargins .~ .init(all: Styles.grid(3))
-
-    _ = self.layer
-      |> checkoutLayerCardRoundedStyle
-      |> \.backgroundColor .~ UIColor.ksr_white.cgColor
-      |> \.shadowColor .~ UIColor.ksr_black.cgColor
-      |> \.shadowOpacity .~ 0.12
-      |> \.shadowOffset .~ CGSize(width: 0, height: -1.0)
-      |> \.shadowRadius .~ CGFloat(1.0)
-      |> \.maskedCorners .~ [
-        CACornerMask.layerMaxXMinYCorner,
-        CACornerMask.layerMinXMinYCorner
-      ]
-
-    _ = self.rootStackView
-      |> verticalStackViewStyle
-      |> \.spacing .~ Styles.grid(3)
-
-    _ = self.titleAndAmountStackView
-      |> self.titleAndAmountStackViewStyle
-
-    _ = self.titleLabel
-      |> self.titleLabelStyle
-
-    _ = self.amountLabel
-      |> self.amountLabelStyle
-
-    _ = self.continueButton
-      |> greenButtonStyle
-      |> UIButton.lens.title(for: .normal) %~ { _ in Strings.Continue() }
+    self.layoutMargins = .init(all: Styles.grid(3))
+
+    self.layer.cornerRadius = 16.0
+    self.layer.backgroundColor = UIColor.ksr_white.cgColor
+    self.layer.shadowColor = UIColor.ksr_black.cgColor
+    self.layer.shadowOpacity = 0.12
+    self.layer.shadowOffset = CGSize(width: 0, height: -1.0)
+    self.layer.shadowRadius = CGFloat(1.0)
+    self.layer.maskedCorners = [
+      CACornerMask.layerMaxXMinYCorner,
+      CACornerMask.layerMinXMinYCorner
+    ]
+
+    self.rootStackView.axis = NSLayoutConstraint.Axis.vertical
+    self.rootStackView.spacing = Styles.grid(3)
+
+    self.titleAndAmountStackView.backgroundColor = .ksr_white
+    self.titleAndAmountStackView.layoutMargins = UIEdgeInsets(leftRight: Styles.gridHalf(4))
+
+    self.titleLabel.accessibilityTraits = UIAccessibilityTraits.header
+    self.titleLabel.adjustsFontForContentSizeCategory = true
+    self.titleLabel.font = UIFont.ksr_headline(size: 15)
+    self.titleLabel.numberOfLines = 0
+    self.titleLabel.text = Strings.Total_amount()
+
+    self.amountLabel.adjustsFontForContentSizeCategory = true
+    self.amountLabel.textAlignment = NSTextAlignment.right
+    self.amountLabel.isAccessibilityElement = true
+    self.amountLabel.minimumScaleFactor = 0.75
+
+    self.continueButton.setTitle(Strings.Continue(), for: .normal)
+    self.continueButton.setTitleColor(.ksr_white, for: .normal)
+    self.continueButton.setTitleColor(.ksr_white, for: .highlighted)
+    self.continueButton.setBackgroundColor(.ksr_create_700, for: .normal)
+    self.continueButton.setBackgroundColor(UIColor.ksr_create_700.mixDarker(0.36), for: .highlighted)
+    self.continueButton.setBackgroundColor(UIColor.ksr_create_700.mixLighter(0.36), for: .disabled)
+    self.continueButton.clipsToBounds = true
+    self.continueButton.layer.masksToBounds = true
+    self.continueButton.layer.cornerRadius = Styles.grid(2)
```

Comment:
I don't think this is the right way to move away from prelude. This copy-pastes code from `baseButtonStyle` and `greenButtonStyle`. In the long run, I think we want to convert these common styles into a different kind of utility method (ex static functions that take the button as input or maybe extensions on the UIButton class). For now, however, I think we should still use the prelude helpers, and just switch the additional styling unique to this class to standard UIKit. If we copy-paste it like this, we'll probably lose track of this by the time we switch the green button style, and it'll be more of a hassle to change that in the future (ex when we add dark mode support).

So I love the idea here and I think any styles that were just defined in this class (or prelude used directly) is worth getting rid of, but I don't want us to start copy-pasting common styling throughout the app.